### ISSUE TITLE
Optional fields (>= proto 3.15) are not supported by the code generator

### ISSUE THREAD

**Issue by brphelps** [twitchtv/twirp-ruby#71]
This happens when creating a proto property that is `optional`, and trying to generate code.

```
<yourProto>.proto: is a proto3 file that contains optional fields, but code generator protoc-gen-twirp_ruby hasn't been updated to support optional fields in proto3. Please ask the owner of this code generator to support proto3 optional.--twirp_ruby_out: 
```

**Comment by PatriciaHudakova** [comment]
This is a real issue... I really hope there is a fix for this soon...

**Comment by wmatveyenko** [comment]
This was fixed in #79
